webapp/src/test/java/com/box/l10n/mojito/metrics/StatsdNamingConventionConfigTest.java

```diff
@@ -0,0 +1,56 @@
+package com.box.l10n.mojito.metrics;
+
+import org.junit.jupiter.api.Test;
+
+import io.micrometer.core.instrument.config.NamingConvention;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.junit.jupiter.api.Assertions.*;
+
+class StatsdNamingConventionConfigTest {
```

Comment:
knit: Thank you for adding test, this is great. Usually i'm fine with no test for plain POJO with no business logic, including `@ConfigurationProperites`

Reply:
I agree, I tend not to add tests for units without business logic, but this one has the `changesNaming` method and I wanted to make sure it was doing what was intended.